bin/minimize_vcf.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3.8
+
+import pandas as pd
+import sys
+import io
+
+def main():
+    inputvcf = sys.stdin.read()
+    df = pd.read_csv(io.StringIO(inputvcf), sep="\t", comment="#", header=None)
+    df = df.rename(columns={
+        0: '#CHROM',
+        1: 'POS',
+        2: 'ID',
+        3: 'REF',
+        4: 'ALT',
+        5: 'QUAL',
+        6: 'FILTER',
+        7: 'INFO',
+        8: 'FORMAT',
+    })
+    df[["INFO"]] = "."
+    df = df[df.FILTER == "PASS"]
+    vcfbody = df.to_csv(sep="\t", index=False)
+
+    vcfformatheader = "\n".join(line for line in inputvcf.splitlines() if line.startswith("##FORMAT"))
+    vcfcontigheader = "\n".join(f"##contig=<ID={chr},length={gdf.POS.max()}>" for (chr, gdf) in df.groupby("#CHROM"))
```

Comment:
please change to a predefined one, and we also need to process per different genome version.

Reply:
Actually I tried to avoid to use `*.fa` file in this step for simplicity, and the contig part would not be used after VEP process.
If we are supposed to make contig data correctly using `*.fa`, I think we have more proper approaching like `bcftools reheader --fai ref_genome.fai input.vcf > output.vcf` than simplifying the header information.